run_targets.Rmd

```diff
@@ -71,12 +76,19 @@ library(targets)
 tar_make()
 ```
 
-Errors are okay in the pipeline, they are by design. Only errors that prematurely end the pipeline need to be addressed. Check the network diagram in the next section to confirm you've successfully run the pipeline.
+Only errors that prematurely 
+end the pipeline need to be addressed. Check the network diagram in the next section 
+to confirm you've successfully run the pipeline. As long as your dialog box reads 'end pipeline' 
+after 'built pattern eeRun', this workflow has successfully completed.
 
 ### Create a network diagram of the workflow.
 
 ```{r show_vis}
 tar_visnetwork()
 ```
 
-Depending on how your config file is set up, you may have a few red circles on this graph. The important part is that the square on the right hand side is either orange or green.
+Depending on how your config file is set up, you may have a few red circles (errors) on this graph. 
+Errors for `polygons`, `centers`, and `locs` are okay in the pipeline, they are by design: these 
```

Comment:
This text helps a lot!

Reply:
YAY! I find it really helpful to know what isn't clear, because in my head it's crystal - and I might think I wrote out explanations... but in this case they are all in the _targets.R file that most end users would never look at. lol. Thanks for keeping me accountable.